### ISSUE TITLE
specify transport

### ISSUE THREAD

**Issue by Katherine-Li** [socketio/socket.io-client-cpp#71]
I am not sure if it is the right place to ask a question here:
Assume that on my server side I support only polling, (nodejs)
var io = require('socket.io')(server,{transports:(['polling'])});
how can I specify in the socket-io-client-cpp that I would like to connect to server using "polling"?
When I use 
Sio::Client h;
h.connect("http://127.0.0.1:80");

This seems to use "websocket". In this case an underlying transport error is reported.

**Comment by idubinets** [comment]
This is old issue, but I also came with idea to use socket.io c++ library. I need http fallback supporting when Websockets are blocked by firewall. Is it possible? does this library support fallbacks?

**Comment by jmigual** [comment]
Closing as duplicate of #251
